### ISSUE TITLE
Tabs being eaten. Unknown.

### ISSUE THREAD

**Issue by jaunkst** [angular/material#5770]
Very strange behavior in  tabs  v1.0.0-rc4-master-9d52697

below does not work
One of the tabs seems to not show/go behind other tabs or something.
````
openTabs = [{title: 'LEADS'}, {title: 'LEADS'}, {title: 'LEADS'}]
````

below works find.
````
openTabs = [{title: 'LEAD1'}, {title: 'LEAD2'}, {title: 'LEAD3'}]
````

When testing with various random tab titles sometimes one will get eaten up and not show up until one of the tabs are removed dynamically.

This also happens when hard-coded
````

md-tabs(style='width: 500px;')
  md-tab
    md-tab-label
      | Leads
  md-tab
    md-tab-label
      | Leads
  md-tab
    md-tab-label
      | Leads


````

**Comment by jaunkst** [comment]
I see that the md-pagination-wrapper can have the wrong width causing the tabs to wrap.
md-pagination-wrapper

**Comment by jaunkst** [comment]
![cursor_and_arxia_and_2__electron____electron_helper_](https://cloud.githubusercontent.com/assets/982066/11195110/b6f99530-8c75-11e5-9c50-23a7006383fd.png)

**Comment by jaunkst** [comment]
starting the width at 2 instead of 1 corrects the issue for me. 
````
  function updatePagingWidth() {
    var width = 2;
    angular.forEach(getElements().dummies, function (element) { width += element.offsetWidth; });
    angular.element(elements.paging).css('width', width + 'px');
  }
````

**Comment by gpopovic** [comment]
+1

**Comment by scottreisdorf** [comment]
This is still not fixed in  v1.0.0-rc7, See my code Pen:  http://codepen.io/scottreisdorf/pen/EPjXzN
and related Issue I just made: https://github.com/angular/material/issues/6212
